mcp-server/src/templates/prompts/work_issue.yaml

```
@@ -140,6 +140,18 @@ template: |
   
   {{> quality-checks}}
   
+  {{#unless features.workflow.autoCommit}}
+  ## Manual Testing Checkpoint
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)